packages/core/src/client.test.ts

```diff
@@ -1041,6 +1067,27 @@ test('graphql variables', async () => {
   expect(body.variables).toBeDefined();
 });
 
+test('Auto batch single request', async () => {
+  const medplum = new MedplumClient({ ...defaultOptions, autoBatchTime: 100 });
+  const patient = await medplum.readResource('Patient', '123');
+  expect(patient).toBeDefined();
+});
+
+test('Auto batch multiple requests', async () => {
```

Comment:
Would be good to add a few  tests for non-`readResource` cases (e.g readHistory, readReference)